### PR TITLE
Check controller ports

### LINKED ISSUE #125: Show the status of controllers on the shuffleboard
not sure if possible but it would be cool to show whether the correct controllers are plugged in

### PR DESCRIPTION
**Pull request by maxh65000**
Fixes #125

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -119,8 +119,7 @@
   public void robotInit() {
     // Zero all nesscary sensors on Robot
     Config.getInstance().reload();
-
```
Comment (ajwharton) [inline src/main/java/frc/robot/Robot.java:122]: Did you mean to remove the logging code?

File: src/main/java/frc/robot/OI/XboxController.java [hunk 2]
```diff
@@ -46,6 +47,15 @@
   // Pass in the port of the Controller
   public XboxController(int portArg) {
     mController = new Joystick(portArg);
+    checkNameAndPort();
+  }
+
+  public boolean checkNameAndPort(){
+    if(!mController.getName().equals("Controller (Xbox One For Windows)") || mController.getPort() != 0){
```
Comment (ajwharton) [inline src/main/java/frc/robot/OI/XboxController.java]: This string feels like a great constant

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (maxh65000) [reply src/main/java/frc/robot/Robot.java:122]: Yes

File: src/main/java/frc/robot/OI/XboxController.java (hunk 2 above)
Comment (maxh65000) [reply src/main/java/frc/robot/OI/XboxController.java]: Fixed

**Review by ajwharton — APPROVED** [review]

**Pull request merged**